yellowbrick/features/manifold.py

```diff
@@ -314,9 +314,9 @@ def fit_transform(self, X, y=None):
             y = np.asarray(y)
             self.range_ = (y.min(), y.max())
 
-        start = time.time()
-        Xp = self.manifold.fit_transform(X)
-        self.fit_time_ = time.time() - start
+        with Timer() as timer:
+            Xp = self.manifold.fit_transform(X)
+        self.fit_time_ = timer.interval
```

Comment:
This is great, thank you so much for updating this! One thing I was wondering, what would happen if we did this as follows:

```python
with Timer() as self.fit_time_:
    Xp = self.manifold.fit_transform(X) 
```

This would allow the user to `print(self.fit_time_)` to get the human readable interval, but we would probably have to update the title of the figure to ensure it worked correctly.

Reply:
`self.fit_time_` will then be an object instead of the string. 

We will have to change the documentation [here](https://github.com/DistrictDataLabs/yellowbrick/blob/43405974e25fa600b38aa9a7e52a6e86d0bd0e59/yellowbrick/features/manifold.py#L154) and the plotting part [here](https://github.com/DistrictDataLabs/yellowbrick/blob/43405974e25fa600b38aa9a7e52a6e86d0bd0e59/yellowbrick/features/manifold.py#L398).

Let me know if there are other changes that are needed.